mtgjson4/compile_mtg.py

```diff
@@ -75,12 +77,35 @@ def build_output_file(sf_cards: List[Dict[str, Any]], set_code: str) -> Dict[str
 
     LOGGER.info("Starting tokens for {}".format(set_code))
     sf_tokens: List[Dict[str, Any]] = scryfall.get_set("t" + set_code)
-    output_file["tokens"] = build_mtgjson_tokens(sf_tokens)
+    output_file["tokens"] = build_mtgjson_tokens(sf_tokens + added_tokens)
     LOGGER.info("Finished tokens for {}".format(set_code))
 
     return output_file
 
 
+def transpose_tokens(
+    cards: List[Dict[str, Any]]
+) -> Tuple[List[Dict[str, Any]], List[Dict[str, Any]]]:
+    """
+    Sometimes, tokens slip through and need to be transplanted
+    back into their appropriate array. This method will allow
+    us to pluck the tokens out and return them home.
+    :param cards: Cards+Tokens to iterate
+    :return: Cards, Tokens as two separate lists
+    """
+    # Order matters with these, as if you do cards first
+    # it will shadow the tokens lookup
+    tokens = [
+        scryfall.download(scryfall.SCRYFALL_API_CARD + card["uuid"])
+        for card in cards
+        if card["layout"] == "token"
+    ]
+
+    cards = [card for card in cards if card["layout"] != "token"]
```

Comment:
Wouldn't it be more efficient to just change the old list?
like:
```
indexes = [i for i, card in enumerate(cards) if card == "token"]
tokens = [cards.pop(i) for i in indexes[::-1]]
```

Reply:
The list is no more than 500 elements, so it shouldn't be a big deal. 

I'm also not 100% certain what your iteration entails... looks like you're reverse-iterating the cards array and popping at the found index... I don't believe you can remove from a list while iterating